demos/demo-minimal-js/index.js

```diff
@@ -225,18 +253,39 @@ export function userUnavailable() {
   enableButtons([USER_AVAILABLE]);
 }
 
-export function incomingCall() {
+export function incomingCall(optionalPayload) {
   state.externalCallId = uuidv4();
+  const payload = {
+    createEngagement: true,
+    fromNumber: state.fromNumber,
+    toNumber: state.toNumber,
+    externalCallId: state.externalCallId,
+  };
   window.setTimeout(() => {
-    cti.incomingCall({
-      createEngagement: true,
-      fromNumber: state.fromNumber,
-      toNumber: state.toNumber,
-      externalCallId: state.externalCallId,
-    });
+    cti.incomingCall(optionalPayload || payload);
   }, 500);
   disableButtons([OUTGOING_CALL, INCOMING_CALL, USER_UNAVAILABLE]);
   enableButtons([ANSWER_CALL, END_CALL]);
+
+  // Trigger incoming call from window
+  // Send message to all open remote tabs
+  if (!state.usesCallingWindow && state.iframeLocation === "window") {
+    bc.postMessage({ type: INCOMING_CALL, payload });
+  }
```

Comment:
This allows us to trigger an `INCOMING_CALL` event in all open tabs from the calling window